src/dex/verified/pools/secondary/secondaryPoolMath.ts

```diff
@@ -0,0 +1,592 @@
+import { BI_MAX_INT, BI_POWS } from '../../../../bigint-constants';
```

Comment:
@Mohzcrea8me if math functions are utility functions shared by primary and secondary pools, perhaps we can have one common poolMath.ts above primary and secondary pool folders ?

Reply:
Yes thought of doing this, verified pool testing took my time I didn't get to run full code clean all these will be corrected